rocky/reports/report_types/safe_connections_report/report.html

```diff
@@ -48,9 +48,9 @@ <h4 id="safe-connections-{{ type }}">{{ type }} {% translate "server" %}</h4>
     </div>
     {% for ip, findings in data.sc_ips.items %}
         {% if findings %}
-            <h5>{{ ip }}</h5>
+            <h5>{% translate "Host: " %}{{ ip.tokenized.address }}</h5>
```

Comment:
Only translate actual text
```suggestion
            <h5>{% translate "Host" %}: {{ ip.tokenized.address }}</h5>
```